### ISSUE TITLE
imread error while importing mglearn

### ISSUE THREAD

**Issue by hianjana** [amueller/mglearn#8]
I'm getting "ImportError: cannot import name 'imread'" when I try to import mglearn
As per a previous thread, I have tried this: 
           from scipy.misc.pilutil import imread
But the error is still there. Please help.

**Comment by amueller** [comment]
you need to install pillow.

**Comment by hianjana** [comment]
pillow is already installed. Still it is giving the same error.

**Comment by amueller** [comment]
Is pillow installed in that environment? If ``from scipy.misc.pilutil import imread
`` gives an error that means it's unrelated to mglearn and there's an issue with the PIL/Pillow installation.

**Comment by hianjana** [comment]
I do get an error when I try: from scipy.misc.pilutil import imread
         It says: No module named 'Image'
So I did: import Image
         Now it says: No module named 'Image'
I also did: import PIL
         It says: No module named 'PIL'
So I tried to install PIL again by: pip install Pillow 
I also tried: python -m pip install Pillow
         I get a message: "Requirement already satisfied: Pillow in c:\python36\lib\site-packages (5.2.0)"
It looks like Pillow is installed to me. Is there something else I can do to fix this?

**Comment by amueller** [comment]
How are you executing the code?
Whether you're using Jupyter or not, this article might be helpful:
https://jakevdp.github.io/blog/2017/12/05/installing-python-packages-from-jupyter/

**Comment by ebunt** [comment]
you can install imageio.
```bash
pip install imageio
```
**replace imread in the these 2 files**
1. [Plot animal tree](https://github.com/amueller/mglearn/blob/master/mglearn/plot_animal_tree.py)
2. [Plot interactive tree](https://github.com/amueller/mglearn/blob/master/mglearn/plot_interactive_tree.py)

**with this**

```python
from imageio import imread
```
3. add imageio to the install_requires list in the setup.py

```python
python setup.py install
```

**Comment by amueller** [comment]
should be fixed now.

**Comment by radlyeel** [comment]
Today (6/15/2023)) I downloaded the support code for the OReilly book, and ran into this error from Jupyter notebook.

File ~/Dropbox/labs/machine_learning/Mueller/introduction_to_ml_with_python/mglearn/plot_interactive_tree.py:8
      6 from io import StringIO
      7 from sklearn.tree import export_graphviz
----> 8 from imageio import imread
      9 from scipy import ndimage
     10 from sklearn.datasets import make_moons

ModuleNotFoundError: No module named 'imageio'  

I killed the Notebook server and launched Python(3.11.3) directly, where I did "from imageio import imread" with no problem.  Jupyter had been launched from the same directory.  If I create a new notebook and do 'from imageio import imread' in the first cell, it's okay.  Any suggestions?
